### DISCUSSION TITLE
Gaomon S620 not working with lazer drivers

### DISCUSSION THREAD

**Discussion by itsnobii** [ppy/osu#13652] [Q&A] [answered]
Hey peeps, I've seen the Gaomon S620 work for others on lazer with the new built-in drivers, but for me, the game just says "No tablet detected." No other tablet drivers are running on my computer atm.

![{D6E497C1-A14C-4B52-98A2-B562FC54BB36}](https://user-images.githubusercontent.com/57056220/123351326-dbecbd00-d52a-11eb-9792-47590f23c616.png)

Maybe I'm missing something incredibly obvious but I can't seem to figure it out.
Any help would be appreciated!

**Comment by frenzibyte** [comment, marked as answer]
Check [OpenTabletDriver's Windows FAQ](https://github.com/OpenTabletDriver/OpenTabletDriver/wiki/Windows-FAQ) (assuming you're on Windows, you haven't stated your OS), perhaps [this section](https://github.com/OpenTabletDriver/OpenTabletDriver/wiki/Windows-FAQ#my-gaomonhuion-tablet-isnt-detected-but-is-listed-as-supported) is precisely the one for you.

**Reply by itsnobii** [reply]
Thank you sm for the help!
